src/commands/Fun/destruct.js

```diff
@@ -0,0 +1,64 @@
+exports.run = function(bot,msg,args) {
+
+    let parsedArgs = bot.utils.parseArgs(args, ['d:','s:']);
+
+    if(parsedArgs.leftover.length < 1) {
+        throw 'Please provide a secret message';
+    }
+
+    let text = parsedArgs.leftover;
+    let message = text.join(' ');
+    let delay = isNaN(parsedArgs.options.d) ? 5000 : parsedArgs.options.d;
```

Comment:
You should call parseInt on the option value. After that, you should add a check to make sure that the value is not negative.